subscription/notify-lambda/build.sh

```diff
@@ -0,0 +1,21 @@
+# This script is used by the bamboo build project.
+
+#!/bin/bash
```

Comment:
I don't think this matters at all for zsh but you should have shebang as the very first line. Something lurking deep in un*x may care and only when you need it most.

Reply:
I put the shebang on the first line.